README.md

```diff
@@ -79,12 +82,15 @@ The following options should be set in your `terraform.tfvars` file (or entered
 
 |Option|Type|Description|Updatable|
 |:-----|:---|:----------|:-------:|
+| `a_record_name` | string  | Name of the A record where the ecl watch ip is placed when you provide a dns zone | Y |
```

Comment:
Add text noting that this is optional, like the other optional items here.

Also, not everyone is technical enough to know what a DNS A record is.  Maybe reword this, or add an explanation and example?